acceptance/indy_acceptance/indy-tests/test_results/Acceptance_Test_20171117_17-42-58/Test_scenario_04_Keyrings_Wallets_20171117_17-42-58.json

```diff
@@ -0,0 +1,28 @@
+{
+  "testcase": "Test_scenario_04_Keyrings_Wallets",
```

Comment:
It looks like a log. I think it needs to be removed

Reply:
We removed it.